### PR TITLE
fix result=null case & add error handle & fix txHash search query

### PR DESCRIPTION
**Pull request by 21paradox**
fix result=null case & add error handle & fix txHash search query

**What kind of change does this PR introduce?**

**Did you add tests for your changes?**

**Does this PR introduce a breaking change?**

**What needs to be documented once your changes are merged?**

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/conflux-scan/171)

### CODE REVIEW

File: src/pages/Token/detail.js [hunk 1]
```diff
@@ -120,7 +132,11 @@ class TokenDetail extends Component {
       contractAddress,
     };
     if (filter) {
-      params.address = filter;
+      if (isAddress(filter)) {
+        params.address = tranferToLowerCase(filter);
+      } else {
+        params.txHash = tranferToLowerCase(filter);
+      }
     }
```
Comment (AngeliaQ) [inline src/pages/Token/detail.js:140]: 这个以前只支持搜索address？

File: src/pages/Token/detail.js [hunk 2]
```diff
@@ -353,6 +369,14 @@ class TokenDetail extends Component {
                   <div className="transfer-search-input">
                     {renderAny(() => {
                       const doSearch = () => {
+                        if (!isAddress(searchInput) && !isHash(searchInput)) {
+                          notice.show({
+                            type: 'message-error-light',
+                            content: i18n('Search paremeter incorrect'),
```
Comment (AngeliaQ) [inline src/pages/Token/detail.js]: 拼写错误，parameter，这个文案有没有找产品组确认？

**Review by AngeliaQ — DISMISSED** [review]

File: src/pages/Token/detail.js (hunk 1 above)
Comment (21paradox) [reply src/pages/Token/detail.js:140]: 以前服务端做了处理

File: src/pages/Token/detail.js (hunk 2 above)
Comment (21paradox) [reply src/pages/Token/detail.js]: fixed，文案已更新

**Review by AngeliaQ — APPROVED** [review]

**Pull request merged**